CHANGELOG.md

```diff
@@ -8,6 +8,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Fixed
 - Fixing `ManualKey<0>` to act properly - [#1670](https://github.com/paritytech/ink/pull/1670)
+- Add `call-runtime` support - [#1641](https://github.com/paritytech/ink/pull/1641)
```

Comment:
I think this one should be placed in a separate **### Added** section

Reply:
ahh, sorry, I must have messed it up during merge :( fixed